src/client/sandbox/event/hover.js

```diff
@@ -77,6 +77,6 @@ export default class HoverSandbox extends SandboxBase {
     attach (window) {
         super.attach(window);
 
-        this.listeners.addInternalEventListener(window, ['mouseover', 'touchstart'], e => this._hover(e.target));
```

Comment:
We should listen `window` events for all browsers and add additional condition for safari